chemprop/utils.py

```diff
@@ -528,3 +528,86 @@ def save_smiles_splits(data_path: str,
 
     with open(os.path.join(save_dir, 'split_indices.pckl'), 'wb') as f:
         pickle.dump(all_split_indices, f)
+
+
+def update_prediction_args(predict_args: PredictArgs,
+                           train_args: TrainArgs,
+                           missing_to_defaults: bool = True,
+                           for_molecule_fingerprint: bool = False) -> None:
```

Comment:
Is this argument necessary? It's not specified in the molecule_fingerprint script.

If it is necessary, I would suggest using a more specific name that indicates what it does rather than when it's used, like `validate_feature_arg` or similar.

Reply:
This was an unintentional exclusion from the molecule_fingerprint script, which has now been added.

I think this is a better way to describe the check.